ReactiveCocoa/Swift/Signal.swift

```diff
@@ -154,6 +154,15 @@ public final class Signal<Value, Error: ErrorType> {
 	}
 }
 
+extension Signal {
+	/// Number of observers observing the signal or `nil` if signal is terminated.
+	///
+	/// Property used for Unit Tests.
+	internal var observersCount: Int? {
```

Comment:
It's still exposing internal state.
I'd rather us test this by verifying that the subscriptions have terminated on every observer.